auth_backends/backends.py

```diff
@@ -29,7 +29,12 @@ class EdXBackendMixin(object):
         'given_name': 'first_name',
         'family_name': 'last_name',
         'locale': 'language',
+        # 1. From where is 'user_tracking_id' supposed to come? From OpenID Connect?
+        # - See https://github.com/edx/edx-platform/blob/1dd1d25bc44cff99cec849ff4695bb1dc92ddac1/lms/djangoapps/oauth2_handler/handlers.py#L57
+        # 2. Should this 'user_tracking_id' be removed if it is overwritten in EdXOpenIdConnect?
+        # 3. Even if 'user_tracking_id' is sent, who uses it? Maybe edx-analytics-dashboard?
         'user_tracking_id': 'user_tracking_id',
```

Comment:
This field is set only with OpenID Connect.  So you can remove it from here.  (Already exists in `EdXOpenIdConnect`.)